### DISCUSSION TITLE
life/love/

### DISCUSSION THREAD

**Discussion by giscus** [xueweiwujxw/xww_blog#10] [Announcements]
# life/love/

她真的太可爱啦

https://wlanxww.com/life/love/

**Comment by sincerely-hello-world** [comment]
路过来看看 博主的 友链是怎么配置的，没想到有瓜吃。doge

**Reply by xueweiwujxw** [reply]
已经是女朋友了，我博客好久没更了，最近搞了一个新玩意儿，准备写个说明，再升级一下FixIt主题
